src/test/java/wooteco/subway/line/LineAcceptanceTest.java

```diff
@@ -21,19 +24,29 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @DisplayName("노선 관련 기능")
-@Transactional
+@Sql("classpath:test.sql")
 public class LineAcceptanceTest extends AcceptanceTest {
+    private static final long EXIST_GANGNAM = 1L;
```

Comment:
```suggestion
    private static final long 강남역_id = 1L;
```
변수명을 좀더 읽기 쉽게 써보면 어떨까요